src/toxcore/binary_io.rs

```diff
@@ -37,9 +37,9 @@ pub trait FromBytes<Output> {
 pub fn array_to_u16(array: &[u8; 2]) -> u16 {
     trace!("Casting array to u16 from array: {:?}", array);
     let mut result: u16 = 0;
-    for byte in 0..array.len() {
+    for pos in 0..array.len() {
```

Comment:
I changed it because you somewhere already used name "pos" and I changed "n" to "pos" below, but it is insignificant - I can remove this change from commit if you would like to.